content/tools/js-libs/http-api-guide.md

```diff
@@ -210,313 +500,593 @@ Here's a simple example that will run `authenticate` for a fakezod when the _Con
+### Delete a channel {#delete-a-channel}
+Rather than just closing individual subscriptions, the entire channel can be closed with the `delete()` function in `Urbit()`. When a channel is closed, all subscriptions are cancelled and all pending updates are discarded. The function takes no arguments, and can be called like `api.delete()`.
 
 ### Reset {#reset}
-
-An existing instance of `http-api`'s `Urbit` class can be reset with its `reset` function. This function takes no arguments, and can be called like `api.reset()` (replacing `api` with whatever you've named the `Urbit` object you previously instantiated). When called, the channel will be closed and all subscriptions cancelled. Additionally, all outstanding outbound pokes will be discarded and a fresh channel ID will be generated.
+An existing `Urbit()` object can be reset with its `reset()` function. This function takes no arguments, and can be called like `api.reset()`. When a channel is reset, all subscriptions are cancelled and all pending updates are discarded. Additionally, all outstanding outbound pokes to the agent will be discarded, and a fresh channel ID will be generated.
 
 ## Further reading {#further-reading}
 
-- [Eyre External API Reference][eyre-ext-ref] - Lower-level documentation of Eyre's external API.
-- [Eyre Guide][eyre-guide] - Lower-level examples of using Eyre's external API with `curl`.
-- [`http-api` on Github][http-api-src] - The source code for `@urbit/http-api`.
-
-## Examples {#examples}
-
-Here are some examples which make use of `@urbit/http-api` that might be useful as a reference:
-
-- [Bitcoin Wallet](https://github.com/urbit/bitcoin-wallet)
-- [Web Terminal](https://github.com/urbit/urbit/tree/master/pkg/interface/webterm)
-- [UrChatFM](https://github.com/urbit/urbit-webrtc/tree/master/urchatfm)
-
-[eyre-ext-ref]: /system/kernel/eyre/reference/external-api-ref
-[eyre-guide]: /system/kernel/eyre/guides/guide
-[http-api-src]: https://github.com/urbit/js-http-api
-[eyre]: /glossary/eyre
-[vane]: /glossary/vane
-[arvo]: /glossary/arvo
-[hoon]: /glossary/hoon
-[gall]: /glossary/gall
-[clay]: /glossary/clay
-[desk]: /glossary/desk
-[mark]: /glossary/mark
-[sse]: https://html.spec.whatwg.org/#server-sent-events
-[@urbit/http-api]: https://github.com/urbit/urbit/tree/master/pkg/npm/http-api
+- [`@urbit/http-api` on Github](https://github.com/urbit/js-http-api) - The source code for the JS HTTP API package.
+- [Eyre External API Reference](./system/kernel/eyre/reference/external-api-ref) - Lower-level documentation of Eyre's external API.
+- [Eyre Guide](./system/kernel/eyre/guides/guide) - Lower-level examples of using Eyre's external API with `curl`.
```

Comment:
These links are broken, how did you fix these? @tinnus-napbus

Reply:
I used a python script but essentially they have to a) be relative links (relative to the location of this document) and b) end in .md unless it's pointing at a directory with a README.md inside. In this case I believe they'd be `../../system/kernel/eyre/reference/external-api-ref.md` and `../../system/kernel/eyre/guides/guide.md`